web/frontend/public/config.js

```diff
@@ -53,4 +53,4 @@ var AWS_CONFIG = {
 var API_BASE_URL = "";
 
 // boolean flag to set amplify on/off for user authentication support
-var AMPLIFY_ENABLED = false;
+var AMPLIFY_ENABLED = true;
```

Comment:
is the authentication disabled previously ?

Reply:
Changing to false , as default is false , Thanks!

Reply:
Either way, this gets overwritten automatically when a build gets pushed to prod